eng/common/scripts/Verify-ChangeLog.ps1

```diff
@@ -17,7 +17,13 @@ if ($ChangeLogLocation -and $VersionString)
 else
 {
   $PackageProp = Get-PkgProperties -PackageName $PackageName -ServiceDirectory $ServiceDirectory
-  $validChangeLog = Confirm-ChangeLogEntry -ChangeLogLocation $PackageProp.ChangeLogPath -VersionString $PackageProp.Version -ForRelease $ForRelease
+  if ($SkipChangeLogVerification.Contains($PackageProp.Name)) {
```

Comment:
What issue is prompting the need for this?  I'd like to better understand if we have other options before we go this route.

Reply:
this is to accommodate [azure-data-nspkg](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/tables/azure-data-nspkg) @scbedd  has conformed that it rightly does not have a CHANGELOG.md